### PR TITLE
RAINCATCH-1172 docs changes

### PR DESCRIPTION
**Pull request by austincunningham**
# What:
Changes to RAINCATCH-1172 made by docs

# Why:
see if we can get tech approval for these changes.

### CODE REVIEW

File: docs/workforce-management-framework/rhmap/master.adoc.orig [hunk 1]
```diff
@@ -0,0 +1,41 @@
+include::topics/shared/attributes.adoc[]
+
+[[WFM-RC-NameLong]]
+= {WFM-RC-NameLong}
+
+[[introducing-raincatcher]]
+== Introducing {WFM-RC-NameLong}
+include::introducing-raincatcher.adoc[leveloffset=+2]
+
+<<<<<<< HEAD
```
Comment (witmicko) [inline docs/workforce-management-framework/rhmap/master.adoc.orig]: git merge conflict

File: docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc [hunk 2]
```diff
@@ -18,4 +18,6 @@ The following features are on the {WFM-RC-NameShort} roadmap:
 * Data Synchronization
 * Integration of sensor data
 
-//Features included in {WFM-RC-NameShort} {WFM-RC-Version} are:
+Features included in {WFM-RC-NameShort} {WFM-RC-Version} are:
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc]: maybe - '{WFM-RC-NameShort} {WFM-RC-Version} features: ' ?

**Review by witmicko — DISMISSED** [review]
minor comments, looks good.
Can you add `*.orig` files to gitignore, or are they needed?

File: docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc (hunk 2 above)
Comment (jstaffor) [reply docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc]: +1

**Comment by jstaffor** [comment]
@witmicko , @austincunningham  - if you can focus on this PR with the goal being that we get this merged by tomorrow COB - it would be a great achievement by all involved!!!

I've looked at it so many times now I cannot see "Can't see the wood for the trees". If there is doc related issues/suggestions - ping me and I'll make the changes after tech approval :+1:

**Comment by witmicko** [comment]
> merged by tomorrow COB

sounds like a solid plan.

**Review by witmicko — APPROVED** [review]
looks good 👍

**Review by jstaffor — APPROVED** [review]

**Pull request merged**